### PR TITLE
make target names more clear

### PR DESCRIPTION
**Pull request by CblPOK-git**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,26 +231,26 @@ Users can generate & inspect intermediate artifacts such as execution trace by r
 
 Unix makefiles:
 ```bash
-make -C ${ZKLLVM_BUILD:-build} circuit_cpp_examples -j$(nproc)
+make -C ${ZKLLVM_BUILD:-build} arithmerics_cpp_example -j$(nproc)
```
Comment (akokoshn) [inline README.md:234]: typo, do you mean `arithmetics_cpp_example`

**Review by akokoshn — CHANGES_REQUESTED** [review]

**Review by akokoshn — APPROVED** [review]

File: .github/workflows/build_linux.yml [hunk 2]
```diff
@@ -151,14 +151,20 @@ jobs:
...
           ls -al ./build/examples/cpp
 
-      - name: Build assigner of the C++ examples
+      - name: Build circuits(.crct) of the C++ examples
         run: |
-          cmake --build build -t assign_cpp_examples
+          cmake --build build -t cpp_examples_generate_crct
+          ls -al ./build/examples/cpp
+
+      - name: Build assignmen tables(.tbl) of the C++ examples
```
Comment (aleasims) [inline .github/workflows/build_linux.yml:164]: Small typo: assignment

File: .github/workflows/build_macos.yml [hunk 3]
```diff
@@ -158,14 +164,20 @@ jobs:
...
           ls -al ./build/examples/rust/target/assigner-unknown-unknown/release/examples
 
-      - name: Build circuits and assignments of the Rust examples
+      - name: Build circuits(.crct) of the Rust examples
         run: |
-          cmake --build build -t assign_rust_examples
+          cmake --build build -t rust_examples_generate_crct
+          ls -al ./build/examples/cpp
+
+      - name: Build assignmen tables(.tbl) of the Rust examples
```
Comment (makxenov) [inline .github/workflows/build_macos.yml:177]: Typo again

**Review by makxenov — APPROVED** [review]

**Comment by aleasims** [comment]
Just a note:

`PRIVATE_INPUT_ARG` is now handling in `add_rust_example_without_proving`, but Rust compiler does not support private inputs now. It's OK, but don't be confused with that, we still don't have private inputs in Rust.

**Pull request merged**